src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx

```diff
@@ -190,15 +192,23 @@ const ProposedSolutionForm: React.FC<ProposedSolutionFormProps> = ({
             ''
           ) : (
             <Box display="flex" flexDirection="row-reverse">
-              <Button
+              <NERFailButton
+                variant="contained"
+                onClick={onClose}
+                form="individual-proposed-solution-form"
+                sx={{ textTransform: 'none', fontSize: 16, marginTop: 3, marginRight: 2 }}
+              >
+                Cancel
+              </NERFailButton>
+              <NERSuccessButton
                 color="success"
                 variant="contained"
                 type="submit"
                 form="individual-proposed-solution-form"
                 sx={{ textTransform: 'none', fontSize: 16, marginTop: 3 }}
               >
-                Add
-              </Button>
+                Submit
+              </NERSuccessButton>
```

Comment:
you only need to replace the Button with the NERSuccessButton and leave it as saying add. No need for the submit/cancel. That's my b, the ticket was poorly worded. Should simplify the code tho/ you shouldn't need to edit those tests (I think)